qutip/core/states.py

```diff
@@ -43,13 +44,29 @@ def _promote_to_zero_list(arg, length):
     raise TypeError("Dimensions must be an integer or list of integers.")
 
 
+def _to_space(dimensions):
+    """
+    Convert `dimensions` to a :class:`.Space`.
+
+    Returns
+    -------
+    space : :class:`.Space`
+    """
+    if isinstance(dimensions, Space):
+        return dimensions
+    elif isinstance(dimensions, list):
+        return Space(dimensions)
+    else:
+        return Space([dimensions])
```

Comment:
What should happen if `dimensions` is already a Dimension object?

Reply:
Create a super operator.